tools/release.sh

```diff
@@ -151,6 +142,7 @@ fi
 mkdir kgs
 kgs="kgs/$version"
 pip freeze | tee $kgs
+pip install pytest --constraint ../constraints.txt
```

Comment:
This command fails now because we've changed directories a couple times and `../constraints.txt` is no longer the right path. To fix this, I think we should just install `pytest` with `josepy` above and delete this line.

If after this change the release script works, this PR LGTM!

Reply:
Ok, let's do this. I tried to avoid to modify the behavior of the release script, even if I feel that some logics could may be simplified. Here is a first step.